airflow/models/baseoperator.py

```diff
@@ -54,14 +54,16 @@
     NotMapped,
 )
 from airflow.models.taskinstance import TaskInstance, clear_task_instances
-from airflow.models.taskmixin import DependencyMixin
 from airflow.sdk.definitions._internal.abstractoperator import AbstractOperator as TaskSDKAbstractOperator
 from airflow.sdk.definitions.baseoperator import (
-    get_merged_defaults as get_merged_defaults,  # Re-export for compat
+    # Re-export for compat
+    chain as chain,
+    chain_linear as chain_linear,
```

Comment:
Should we be emitting deprecation warning so folks update their imports?

Reply:
I was hoping we can do that at "airflow.models" level itself i.e. throw a warning if they import anything from there in the dag code

Reply:
Ah, I see. Sounds good.